wlms/server.go

```diff
@@ -276,7 +327,47 @@ func (server *Server) InjectGamePingerFactory(gpf GamePingerFactory) {
 	server.gamePingerFactory = gpf
 }
 
-func CreateServerUsing(acceptedConnections chan ReadWriteCloserWithIp, db UserDb, messagesIn chan Message, messagesOut chan Message) *Server {
+// Tells the relay server to start a game with the given name
+// The host position in the game is protected by the given password
+func (server *Server) RelayCreateGame(name string, hostPassword string) bool {
+	// Tell relay to host game
+	var success bool
+	var data rpc_data.NewGameData
+	data.Name = name
+	data.Password = hostPassword
+	err := server.relay.Call("RelayRPC.NewGame", data, &success)
+	if err != nil || success == false {
+		log.Fatal("Unable to create a game on the relay server. This should not happen.")
```

Comment:
This brings down the whole metaserver. Do you have such high confidence that this never happens?